doc/source/whatsnew/v0.22.0.txt

```diff
@@ -139,6 +139,8 @@ Other Enhancements
 - :func:`read_excel()` has gained the ``nrows`` parameter (:issue:`16645`)
 - :func:``DataFrame.to_json`` and ``Series.to_json`` now accept an ``index`` argument which allows the user to exclude the index from the JSON output (:issue:`17394`)
 - ``IntervalIndex.to_tuples()`` has gained the ``na_tuple`` parameter to control whether NA is returned as a tuple of NA, or NA itself (:issue:`18756`)
+- :func:`DataFrame.pivot` now accepts a list of values (:issue:`17160`).
```

Comment:
add for the ``values=`` kwargs

Reply:
I'm not sure I get you

Reply:
now accepts a list for the ``values=`` kwarg.